doc/source/about.rst

```diff
@@ -0,0 +1,58 @@
+About
```

Comment:
Can we add a contact email here?

Reply:
Should we use @derNarr's mail as we do it in author mail or should we create a pyndl-dev mailinglist for this?

Reply:
I added @derNarr's mail, but we will probably change to something other later.